airflow/config_templates/config.yml

```diff
@@ -28,6 +28,14 @@ core:
       type: string
       example: ~
       default: "{AIRFLOW_HOME}/dags"
+    dag_bundle_storage_path:
+      description: |
+        The folder where Airflow will store DAG bundles locally.
+        By default, this will be `tempfile.gettempdir()`. This path must be absolute.
```

Comment:
```suggestion
        By default, this will be ``tempfile.gettempdir()``. This path must be absolute.
```

Reply:
Just wondering, should we put things in a subdir instead? (say `Path(gettempdir(), "airflow")`) Since this seems to be the generally accepted convention for Linux tools.

Reply:
Ah good call, will do.